api/token.go

```diff
@@ -535,10 +550,18 @@ func (a *API) setCookieToken(config *conf.Configuration, tokenString string, ses
 	return nil
 }
 
-func (a *API) clearCookieToken(ctx context.Context, w http.ResponseWriter) {
-	config := getConfig(ctx)
+func (a *API) clearCookieTokens(config *conf.Configuration, w http.ResponseWriter) {
+	a.clearCookieToken(config, "accesstoken", w)
```

Comment:
```suggestion
	a.clearCookieToken(config, "access_token", w)
```

Reply:
Maybe `sb-access-token`?

Or even

`Maybe `supabase-refresh-token`? For clarity?